src/state-summary/state-card-content.html

```diff
@@ -42,7 +42,11 @@
   },
 
   _maybeLoadCustomUi: function (stateType) {
-    var isLoaded = this._isLoaded('STATE-CARD-' + stateType.toUpperCase());
+    // Constructor check doesn't work on Safari 10.
+    // Bypassing this check mean that in order to user a built-in
```

Comment:
Yes, custom UI didn't work on Safari 10.
Yesterday someone lend me their browserstack account so I could debug it.

We could remove the isLoaded check altogether. The reason I put it in in the first place was to allow using built-in components as custom UI. I guess it is a very edge-case, and is still possible by placing an empty "custom" html to prevent  loading error.